test/integration/fn_tunnel_cmd.go

```diff
@@ -136,12 +141,59 @@ func validateTunnelCmd(ctx context.Context, t *testing.T, profile string) {
 		return
 	}
 
-	// use FQDN to avoid extra DNS query lookup
-	url = "http://nginx-svc.default.svc.cluster.local."
+	// Get kube-dns Cluster IP
+	c, err := config.Load(profile)
+	if err != nil {
+		t.Errorf("failed to load cluster config: %v", err)
+	}
+	_, ipNet, err := net.ParseCIDR(c.KubernetesConfig.ServiceCIDR)
+	if err != nil {
+		t.Errorf("failed to parse service CIDR: %v", err)
+	}
+	ip, err := util.GetDNSIP(ipNet.String())
+	if err != nil {
+		t.Errorf("failed to get kube-dns IP: %v", err)
+	}
+	dnsIP := "@" + ip.String()
+
+	// Test.2-1 DNS resolution by dig (experimental): https://minikube.sigs.k8s.io/docs/handbook/accessing/#dns-resolution-experimental
```

Comment:
instead of adding comment test1
please use a t.Run("serial"){ } with 3 subtests, that run in a for loop. (to make sure they are in order. like this test
https://github.com/kubernetes/minikube/blob/100ba9b8d3464d22cffd9e02ab46b7a81c82d2ae/test/integration/pause_test.go#L38

Reply:
Thank for your advice!
I'll look it.